lib/client/signalflow/signalflow_client.js

```diff
@@ -74,6 +74,52 @@ function SignalflowClient(apiToken, options) {
     };
   }
 
+  function liveTailRequest(opts) {
+    var msgBuffer = [];
+    var callback = null;
+
+    function resolveMessage(msg) {
+      if (msg.type === 'error') {
+        errorMessageCallback(msg);
+      } else {
+        callback(null, msg);
+      }
+    }
+
+    function errorMessageCallback(msg) {
+      callback(msg, null);
+    }
+
+    function msgCallback(msg) {
+      if (!callback) {
+        msgBuffer.push(msg);
+      } else {
+        resolveMessage(msg);
+      }
+    }
+
+    var callbacks = {
+      onMessage: msgCallback,
+      onError: errorMessageCallback
+    };
+
+    var requestId = rm.registerLiveTail(opts, callbacks, null);
+
+    return {
+      close: function () {
+        return rm.stopLiveTail(requestId);
+      },
+      stream: function (fn, errorFn) {
```

Comment:
I think `errorFn` doesn't end up being used (this is also a mistake in the `execute` path), and instead we do everything through the single callback function `fn` taking two arguments: `fn(error, msg)`.

If we are following the same pattern here, then we don't need this `errorFn` parameter.

Reply:
removed it